lib/verdict/experiment.rb

```diff
@@ -21,6 +21,7 @@ def initialize(handle, options = {}, &block)
     @segmenter                   = options[:segmenter]
     @subject_type                = options[:subject_type]
     @disqualify_empty_identifier = options[:disqualify_empty_identifier]
+    @timestamps_out_of_band      = options[:timestamps_out_of_band]
```

Comment:
Not against in principle, but I think that might actually make it more obscure. I think the good thing about making it as an option makes it clear that you can just ignore it if it's not relevant to what you're doing if you just want the base verdict. 

There's a lot of variables that have default implementations like [this](https://github.com/Shopify/verdict/pull/16/files#r51295257) that I couldn't put in context and didn't know if I should be thinking about.